pkg/cmd/cli/cmd/build.go

```diff
@@ -181,6 +181,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().BoolVarP(&(cfg.ForceCopy), "copy", "c", false, "Use local file system copy instead of git cloning the source url")
 	buildCmd.Flags().StringVar(&(cfg.RuntimeImage), "runtime-image", "", "Image that will be used as the base for the runtime image")
 	buildCmd.Flags().VarP(&(cfg.RuntimeArtifacts), "runtime-artifact", "a", "Specify a file or directory to be copied from the builder to the runtime image")
-
+	buildCmd.Flags().StringVarP(&(cfg.Network), "network", "", "", "Specify the default Docker Network name to be used in build process")
```

Comment:
Hum, I don't think so. cfg.DockerNetworkMode points to the default 'bridge' config. There are also some references in there to --net=container:id in the code. I couldn't understand what's that about (maybe something from Openshift build process), but the case here is that the cfg.DockerNetworkMode is used somewhere that cannot be configured :(

So what I did here is set a new --network flag that would be used only if that's explicitly called by the build process, otherwise the default behaviour would be keept.